### ISSUE TITLE
Autostart. Just 3 empty terminals

### ISSUE THREAD

**Issue by AGA058** [alexa/alexa-avs-sample-app#1072]
I used the headless autostart thread to start alexa without a monitor but after the reboot, there are only 3 empty black Terminal Windows. Is there a possibility to find the Problem?

Greetings and thank you!

**Comment by mradulan** [comment]
Hi @AGA058 , did you try from [this](https://github.com/shivasiddharth/headless-alexa-avs-sample-app) script?

**Comment by AGA058** [comment]
Yeah, i also used this Version. But i only can hear the start voice. More then once, but Alexa did not start..

**Comment by kclchan** [comment]
Hi @AGA058, sorry the script is not part of alexa-avs-sample-app so we won't be able to provide much insights for the issue you are seeing.  You can look at #985 and see if it helps.  Thanks!
